packages/amplify-cli/src/commands/push.ts

```diff
@@ -52,6 +53,13 @@ const syncCurrentCloudBackend = async (context: $TSContext): Promise<void> => {
   }
 };
 
+/**
+ * Updates tracked files for auto updates in the build directory that will not be detected for 'amplify push'
+ */
+const updateTrackedFiles = async (): Promise<void> => {
```

Comment:
is this wrapper method needed? can we just use `await updateCognitoTrackedFiles();` directly?

Reply:
Yes, I figured this could be updated to add each category since they all probably have different ways of changing "tracked" files.